dist/src/main/resources/api/backup-management-api.yaml

```diff
@@ -109,6 +110,99 @@ paths:
           description: Request from gateway to broker timed out
           $ref: '#/components/responses/Error'
 
+  # Section on historic backups
+  /backup-history:
+    post:
+      summary: Takes a historic backup
+      description: Start taking a backup with the given backupId
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TakeBackupRequest'
```

Comment:
:question: Do these components used as response types really match what Operate and Tasklist return? https://docs.camunda.io/docs/next/self-managed/operational-guides/backup-restore/operate-tasklist-backup/#create-backup-api suggests otherwise